.travis.yml

```diff
@@ -15,27 +15,25 @@ git:
 
 env:
   global:
-    - DOCKER_ORG=zencash
-    - IMAGE_NAME=sc-ci-base
     # Separate branches by space
     - PROD_RELEASE_BRANCHES='master'
-    - PUBLISH_BUILD=false
 
-script:
+before_script:
   - source ci/setup_env.sh
   - ci/travis_keep_alive.sh
-  - ci/script.sh tests
 
 jobs:
   include:
-    - name: jdk-11_latest
-      env: PUBLISH_BUILD=true IMAGE_TAG='bionic_jdk-11_latest' TESTS='000'
-
+    - stage: java project
+      name: jdk-11_latest
+      script: ci/script.sh
+      env: DOCKER_ORG='zencash' IMAGE_NAME='sc-ci-base' IMAGE_TAG='bionic_jdk-11_latest' TESTS='000'
```

Comment:
Maybe the list nomenclature could be used here:
```
- stage: java project
      name: jdk-11_latest
      script: ci/script.sh
      env: 
          - DOCKER_ORG='zencash' 
          - IMAGE_NAME='sc-ci-base' 
          - IMAGE_TAG='bionic_jdk-11_latest' 
          - TESTS='000'
```

Reply:
It can be, just more lines, but probably easier to read?

Reply:
I usually prefer multiple lines, easier to read and to find envars